app/models/project.rb

```diff
@@ -74,8 +74,13 @@ class Project < ActiveRecord::Base
 
   pg_search_scope :search_display_name,
     against: :display_name,
-    using: :trigram,
-    ranked_by: ":trigram"
+    using: { tsearch: {
+      prefix: true,
```

Comment:
Could you indent this one more level so it's clearer that it all belongs to tsearch?